Running cluster test times out and locks up my computer

I started working on updating the Debian packaging for libzim and while running tests on 3.2.0+, the cluster test would timeout and lock up my computer for about 30 seconds.

I did a git bisect and tracked it down to 655ff7b6bf4ef7154287231739e53d25cabafb60.

```
km@km-pt:~/projects/libzim/build$ ninja test
[0/1] Running all tests.
1/5 cluster                                 TIMEOUT 30.14 s
2/5 dirent                                  OK       0.01 s
3/5 header                                  OK       0.01 s
4/5 uuid                                    OK       1.01 s
5/5 template                                OK       0.01 s

OK:         4
FAIL:       1
SKIP:       0
TIMEOUT:    1


The output from the failed tests:

1/5 cluster                                 TIMEOUT 30.14 s

--- command ---
MAIL='/var/mail/km' USER='km' COMP_WORDBREAKS=' 	
"'><;|&(:' SHLVL='1' HOME='/home/km' OLDPWD='/home/km/projects' HUSHLOGIN='FALSE' DBUS_SESSION_BUS_ADDRESS='unix:path=/run/user/1000/bus' DEBEMAIL='legoktm@member.fsf.org' DEBFULLNAME='Kunal Mehta' LOGNAME='km' JOURNAL_STREAM='9:1566593' _='/usr/bin/ninja' XDG_SESSION_ID='4' TERM='vt220' PATH='/usr/local/bin:/usr/bin:/bin:/usr/local/games:/usr/games' INVOCATION_ID='d14301e573714059bf85e636ac4c84c5' S_COLORS='auto' XDG_RUNTIME_DIR='/run/user/1000' LS_COLORS='' SHELL='/bin/bash' LC_ALL='C.UTF-8' GPG_TTY='/dev/pts/0' PWD='/home/km/projects/libzim/build' /home/km/projects/libzim/build/test/cluster
--- stdout ---
[==========] Running 6 tests from 2 test cases.
[----------] Global test environment set-up.
[----------] 5 tests from ClusterTest
[ RUN      ] ClusterTest.create_cluster
[       OK ] ClusterTest.create_cluster (0 ms)
[ RUN      ] ClusterTest.read_write_cluster
[       OK ] ClusterTest.read_write_cluster (0 ms)
[ RUN      ] ClusterTest.read_write_empty
[       OK ] ClusterTest.read_write_empty (0 ms)
[ RUN      ] ClusterTest.read_write_clusterZ
[       OK ] ClusterTest.read_write_clusterZ (1 ms)
[ RUN      ] ClusterTest.read_write_clusterLzma
[       OK ] ClusterTest.read_write_clusterLzma (25 ms)
[----------] 5 tests from ClusterTest (26 ms total)

[----------] 1 test from CluterTest
[ RUN      ] CluterTest.read_write_extended_cluster
-------

Full log written to /home/km/projects/libzim/build/meson-logs/testlog.txt
FAILED: meson-test 
/usr/bin/python3 -u /usr/bin/meson test --no-rebuild --print-errorlogs
ninja: build stopped: subcommand failed.
```
Please let me know if there's any other information I can provide.

--- a/test/meson.build
+++ b/test/meson.build
@@ -17,7 +17,7 @@ foreach test_name : tests
                           link_with : libzim,
                           dependencies : deps + [gtest_dep],
                           build_rpath : '$ORIGIN')
-    test(test_name, test_exe)
+    test(test_name, test_exe, timeout : 60)
 endforeach
 
 if get_option('default_library') != 'static'

